SPECS/envoy/envoy.spec

```diff
@@ -0,0 +1,440 @@
+# - https://github.com/envoyproxy/protoc-gen-validate/archive/ab56c3dd1cf9b516b62c5087e1ec1471bd63631e.tar.gz
+# - https://github.com/envoyproxy/sql-parser/archive/b14d010afd4313f2372a1cc96aa2327e674cc798.tar.gz
+# - https://github.com/fmtlib/fmt/archive/6.0.0.tar.gz
+# - https://github.com/gabime/spdlog/archive/v1.4.0.tar.gz
+# - https://github.com/golang/protobuf/archive/v1.4.1.zip
+# - https://github.com/golang/tools/archive/2bc93b1c0c88b2406b967fcd19a623d1ff9ea0cd.zip
+# - https://github.com/google/cel-cpp/archive/80e1cca533190d537a780ad007e8db64164c582e.tar.gz
+# - https://github.com/google/jwt_verify_lib/archive/40e2cc938f4bcd059a97dc6c73f59ecfa5a71bac.tar.gz
+# - https://github.com/google/re2/archive/2020-03-03.tar.gz
+# - https://github.com/googleapis/googleapis/archive/82944da21578a53b74e547774cf62ed31a05b841.tar.gz
+# - https://github.com/gperftools/gperftools/archive/gperftools-2.7.90.tar.gz
+# - https://github.com/grpc-ecosystem/grpc-httpjson-transcoding/archive/faf8af1e9788cd4385b94c8f85edab5ea5d4b2d6.tar.gz
+# - https://github.com/grpc/grpc/archive/d8f4928fa779f6005a7fe55a176bdb373b0f910f.tar.gz
+# - https://github.com/jbeder/yaml-cpp/archive/yaml-cpp-0.6.3.tar.gz
+# - https://github.com/libevent/libevent/archive/0d7d85c2083f7a4c9efe01c061486f332b576d28.tar.gz
+# - https://github.com/lightstep/lightstep-tracer-cpp/archive/3efe2372ee3d7c2138d6b26e542d757494a7938d.tar.gz
+# - https://github.com/mirror/tclap/archive/tclap-1-2-1-release-final.tar.gz
+# - https://github.com/moonjit/moonjit/archive/2.2.0.tar.gz
+# - https://github.com/msgpack/msgpack-c/releases/download/cpp-3.2.1/msgpack-3.2.1.tar.gz
+# - https://github.com/nodejs/http-parser/archive/v2.9.3.tar.gz
+# - https://github.com/opentracing/opentracing-cpp/archive/v1.5.1.tar.gz
+# - https://github.com/openzipkin/zipkin-api/archive/0.2.2.tar.gz
+# - https://github.com/pallets/jinja/archive/2.10.3.tar.gz
+# - https://github.com/pallets/markupsafe/archive/1.1.1.tar.gz
+# - https://github.com/prometheus/client_model/archive/99fa1f4be8e564e8a6b613da7fa6f46c9edafc6c.tar.gz
+# - https://github.com/protocolbuffers/protobuf-go/archive/v1.22.0.zip
+# - https://github.com/protocolbuffers/protobuf/releases/download/v3.10.1/protobuf-all-3.10.1.tar.gz
+# - https://github.com/protocolbuffers/upb/archive/8a3ae1ef3e3e3f26b45dec735c5776737fc7247f.tar.gz
+# - https://mirror.bazel.build/github.com/bazelbuild/platforms/archive/9ded0f9c3144258dad27ad84628845bcd7ca6fe6.zip
+# - https://mirror.bazel.build/github.com/bazelbuild/rules_java/archive/7cf3cefd652008d0a64a419c34c13bdca6c8f178.zip
+# - https://mirror.bazel.build/github.com/golang/protobuf/archive/v1.4.1.zip
+# - https://mirror.bazel.build/github.com/golang/tools/archive/2bc93b1c0c88b2406b967fcd19a623d1ff9ea0cd.zip
+# - https://mirror.bazel.build/github.com/protocolbuffers/protobuf-go/archive/v1.22.0.zip
+Source1:        %{name}-vendor-%{version}.tar.gz
```

Comment:
Please provide instructions on how to generate the vendor tarball in a **reproducible** (see: `protobuf.spec` or `coredns-1.7.0.spec` from `1.0-dev` for instance) way, so that we can later recreate it and know how to generate a new one when we update the version.